lib/Plack/Middleware/EnsThrottle.pm

```diff
@@ -45,6 +45,8 @@ enable 'EnsThrottle::Second',
   },
   blacklist => '192.168.2.1', #blacklist a single IP
   whitelist => ['192.168.0.0-192.169.255.255'], #whitelist a range
+  whitelist_hdr => 'Token',
```

Comment:
Token isn't an official HTTP header. Perhaps it's better to use something more explicit like `X-RateLimit-Token` as the example header.